components/Storyblok/partials/SbLink.tsx

```diff
@@ -109,16 +118,7 @@ export const SbLink = React.forwardRef<HTMLAnchorElement, SbLinkProps>((props, r
   // ---------------------------------------------------------------------------
   if (props.link.linktype === 'asset') {
     // Rewrite the URL to the redirect link to mask the API endpoint.
-    if (config.isNetlify) {
-      linkUrl = linkUrl.replace(
-        /http?(s)\:\/\/a\.storyblok\.com/gi,
-        config.assetCdn + 'a',
-      );
-      linkUrl = linkUrl.replace(
-        /http?(s)\:\/\/img?[0-9]\.storyblok\.com/gi,
-        config.assetCdn + 'i',
-      );
-    }
```

Comment:
Replaced this old code with getMaskedAsset, and I'm just using the masked URL for all the images whether they're Netlify or not